sw/device/tests/usbdev_iso_test.c

```diff
@@ -220,8 +220,13 @@ bool test_main(void) {
       &usbdev_control, ctx->usbdev, 0, config_descriptors,
       sizeof(config_descriptors), test_descriptor, sizeof(test_descriptor)));
 
-  while (usbdev_control.device_state != kUsbTestutilsDeviceConfigured) {
-    CHECK_STATUS_OK(usb_testutils_poll(ctx->usbdev));
+  // Proceed only when the device has been configured; this allows host-side
+  // software to establish communication.
+  CHECK_STATUS_OK(
+      usb_testutils_controlep_config_wait(&usbdev_control, &usbdev));
+
+  if (kDeviceType != kDeviceSimDV && kDeviceType != kDeviceSimVerilator) {
```

Comment:
I suggest creating a new macro wrapping the `LOG_INFO` to perform these checks.

Reply:
Done.